.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "src/rs_driver"]
 	path = src/rs_driver
 	url = https://github.com/RoboSense-LiDAR/rs_driver.git
-	branch = dev_opt
+	branch = main
```

Comment:
Commenting in this section since damn GH doesn't let you comment on submodule changes.

About the rs_driver changes:

I carefully checked all the changed files and couldn't spot many "real" changes so far, the most important ones would be
- RSBP_V4 decoder got merged into RSBP, and they threw a bunch of if/else in the driver to know if it's v3 or v4, The have a non-default variable in the driver, so we need to make sure that this is set to false (will check later) 
![image](https://github.com/botsandus/auto-rslidar-sdk/assets/21349875/b246d346-567d-42e3-b345-590bd155d433)

- Now they have a `reverse` mode, which I have no clue what is, but looks like you can spin the lidar counterclockwise instead of clockwise (or the opposite)
- The new bpearl has a completely different timestamp system:
![image](https://github.com/botsandus/auto-rslidar-sdk/assets/21349875/078492e5-fe20-4124-8918-1cb66f25aba3)